sdk/communication/azure-communication-common/CHANGELOG.md

```diff
@@ -3,12 +3,15 @@
 ## 2.0.0-beta.3 (Unreleased)
 
 ### Features Added
+- Added support for a new communication identifier `MicrosoftTeamsAppIdentifier`.
 
 ### Breaking Changes
+- Removal of `MicrosoftBotIdentifier` and adding of `MicrosoftTeamsAppIdentifier` is a breaking change. Identifiers using rawID starting with `28:orgid:` will be treated as `MicrosoftTeamsAppIdentifier`, other rawIDs starting with `28:` will be treated as `UnknownIdentifier`.
```

Comment:
```suggestion
- Introduction of `MicrosoftTeamsAppIdentifier` is a breaking change. It will affect code that relied on using `UnknownIdentifier` with rawIDs starting with `28:orgid:`, `28:dod:`, `28:gcch:`.
```

Reply:
Updated.